### PR TITLE
Unify headings

### PR DESCRIPTION
**Pull request by datenreisender**
For #380

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -1,66 +1,68 @@
...
+  * [Side Effects May Include...](ch03.md#side-effects-may-include)
+  * [8th Grade Math](ch03.md#8th-grade-math)
+  * [The Case for Purity](ch03.md#the-case-for-purity)
   * [In Summary](ch03.md#in-summary)
 * [Chapter 04: Currying](ch04.md)
-  * [Can't live if livin' is without you](ch04.md#cant-live-if-livin-is-without-you)
-  * [More than a pun / special sauce](ch04.md#more-than-a-pun--special-sauce)
-  * [In summary](ch04.md#in-summary)
+  * [Can't Live If Livin' Is without You](ch04.md#cant-live-if-livin-is-without-you)
+  * [More than a Pun / Special Sauce](ch04.md#more-than-a-pun--special-sauce)
```
Comment (KtorZ) [inline SUMMARY.md]: than -> Than ?

File: ch04.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Chapter 04: Currying
 
-## Can't live if livin' is without you
+## Can't Live If Livin' Is without You
```
Comment (KtorZ) [inline ch04.md]: without -> Without ?

File: SUMMARY.md (hunk 1 above)
Comment (datenreisender) [reply SUMMARY.md]: Nice catch! I will shortly force-push an updated branch with “More Than a Pun / Special Sauce”.

File: ch04.md (hunk 2 above)
Comment (datenreisender) [reply ch04.md]: As described in https://github.com/MostlyAdequate/mostly-adequate-guide/issues/380#issuecomment-364848439 I propose that we keep “without” lowercased.

**Comment by KtorZ** [comment]
Thanks again for the work :+1:

**Pull request merged**